### ISSUE TITLE
3.2.0-RC2 - Strange behavior - X movements past end, bogus printer fan errors

### ISSUE THREAD

**Issue by matthew-humphrey** [prusa3d/Prusa-Firmware#691]
I have been using the RC firmware (3.2.0-RC1) since it was released. And although I like some of the print quality improvements I have been able to get with it, I am seeing some signs of instability. For example, I have seen multiple times where the printer does strange things such as move out on the X axis until it hits the end, and then try to keep going. I also just saw it report a printer fan error, and although it kept printing, it turned off the filament and bed heaters. 

I normally print via Octoprint, however, I just disconnected that and experienced the print fan error printing from the SD card.

**Comment by PavelSindler** [comment]
PR https://github.com/prusa3d/Prusa-Firmware/pull/713 fixes print fan errors. 
Does printer hit the end on rigth side (X max)?

**Comment by matthew-humphrey** [comment]
Yes, right side is where it is hitting. I got the print error 3 times on this same .gcode file. After that, I decided to take the extruder off and check out the fan and the fan wires. Unfortunately, while trying to unload filament, it broke off, so now I have a real problem...

I ohm-ed out the fan connection, wiggled the wires, but it seems there are no problems at all with the fan. So possibly it was a firmware issue. I will check when I get it all re-assembled.
